packages/smoke-tests/tests/smoke-test.js

```diff
@@ -6,34 +6,45 @@ const Login = require('../page-objects/login');
 const AddFriend = require('../page-objects/add-friend');
 const Chat = require('../page-objects/chat');
 const { startServer } = require('../helpers/start-server');
+const { name, repository } = require('../../frontend/package');
+const { getStatus } = require('poll-pr-status');
 
 describe('smoke', function() {
   setUpWebDriver.call(this);
 
   before(async function() {
-    let serverInfo = await startServer();
+    if (process.env.GITHUB_ACTIONS) {
+      let status = await getStatus({
+        repository,
+        context: `netlify/${name}/deploy-preview`
+      });
 
-    this.server = serverInfo.server;
-    this.port = serverInfo.port;
+      this.host = status ? status.target_url : 'https://emberclear.io';
```

Comment:
I think for these, we want to deliberately fail if the target url or status isn't present. 

Reason being, it's possible that someone forgets to update the smoke tests, and their changes would cause the smoke tests to your, but because of a potential fallback to the main URL, no one would know anything in wrong

Reply:
It doesn't ever fallback if it is a PR. `status` is only null if it is on the master branch, post merge.

That does bring up a point though, waiting for the production deploy to https://emberclear.io to finish before running faltest.

Reply:
> It doesn't ever fallback if it is a PR.

I like to be cautious :)

> waiting for the production deploy to https://emberclear.io to finish before running faltest.

maybe the e2e tests can run after the deploy steps on the deployment workflow?

Reply:
Is https://emberclear.io your own custom deploy, or does Netlify handle all of that for you like it does ember-aframe-shim?

Reply:
deploy is done here: https://github.com/NullVoxPopuli/emberclear/blob/master/.github/workflows/frontend-deploy.yml#L40

Reply:
It now is explicit about which target it is testing.